recipes/evotorch/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "evotorch" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/evotorch-{{ version }}.tar.gz
+  sha256: bd1f612ef8709e04d59e197aca618bbe579229b13af205d64ecc671535893d57
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
```

Comment:
Might be good to include those? https://github.com/nnaisense/evotorch/blob/2a6ad9404ff6a1ec625f53d116ea7ed6adab0991/pyproject.toml#L31